launches/launch.object3d.py

```diff
@@ -0,0 +1,73 @@
+from os import path
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+
+launch_path = path.realpath(__file__)
+launch_dir = path.join(path.dirname(launch_path), '..')
+print(launch_dir)
+
+def generate_launch_description():
+
+    launch_path = path.realpath(__file__)
+    launch_dir = path.dirname(launch_path)
+    param_dir = path.join(launch_dir, "param")
+    data_dir = path.join(launch_dir, "data")
+
+    complex_yolo_model = Node(
+        package = 'lidar_detection',
+        executable = 'lidar_detection_node',
+        name = 'lidar_detection_node',
+        parameters=[
+            {'model': 'complex_yolo'},
+            {'device': 'cuda:0'},
+            {'conf_thresh': 0.9}
+        ],
+    )
+
+    mmdetection3d_model = Node(
+        package = 'lidar_detection',
+        executable = 'lidar_detection_node',
+        name = 'lidar_detection_node',
+        parameters=[
+            {'model': 'mmdetection3d'},
+            {'device': 'cuda:0'},
+            {'conf_thresh': 0.4}
+        ],
+    )
+
+    object_viz_deteced_node = Node(
+        package='visualizer',
+        executable='object_visualizer_node',
+        name='object_visualizer_node',
+        parameters=[
+            {'topic': '/detected/objects3d'}
+        ]
+    )
+
+    multi_object_tracker_3d_node = Node(
+        package = 'multi_object_tracker_3d',
+        executable = 'multi_object_tracker_3d_node',
+        name = 'multi_object_tracker_3d_node',
+        parameters=[
+            (path.join(param_dir, "perception/mot3d_params.yaml")),
+        ],
+    )
+
+    object_viz_tracked_node = Node(
+        package='visualizer',
+        executable='object_visualizer_node',
+        name='object_visualizer_node',
+        parameters=[
+            {'topic': '/tracked/objects3d'}
+        ]
+    )
+
+    return LaunchDescription([
+        complex_yolo_model,
```

Comment:
can comment either of the two out right now since we don't want both running at the same time